pkgs/top-level/all-packages.nix

```diff
@@ -9569,7 +9569,7 @@ in
 
   icr = callPackage ../development/tools/icr { };
 
-  scry = callPackage ../development/tools/scry {};
+  scry = callPackage ../development/tools/scry { crystal = crystal_0_35; };
```

Comment:
You need to change the input in https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/scry/default.nix#L1 to crystal.

Reply:
Do you have a reference to where we've decided to use that approach as opposed to just overriding the input? IMHO is less messy to do it this way although it is of course less clear for someone who is just looking at the derivation itself in isolation to figure out what is going on.

Reply:
Ahhh, I had my own patched version of scry here.

Reply:
That's why it was building for me.